src/main/configurations/Translate/Configuration_GeefLijstZaakdocumenten_Lv01.xml

```diff
@@ -13,6 +13,12 @@
                 <Exit name="EXCEPTION" state="ERROR"/>
             </Exits>
 
+            <XmlIfPipe name="CheckApiVersion" xpathExpression="${zaakbrug.zgw.zaken-api.version} &lt; 1.5">
```

Comment:
Not sure if you can reference a property like in the xpathExpression like this. Usually it is needed to reference the property in a param and then reference the param in the xpathExpression. Can you check if setting `zaakbrug.zgw.zaken-api.version` >= 1.5 takes the correct forward?